.pre-commit-config.yaml

```
@@ -18,6 +18,12 @@ repos:
     language: system
     files: >-
       ^setup\.py$
+  - id: docs-rebuild
+    name: Checks that generated docs are in-sync with commit
+    entry: python -m ansiblelint.generate_docs
+    language: python
+    pass_filenames: false
```

Have you tried this?
```suggestion
    pass_filenames: false
    additional_dependencies:
    - .
```

---

I does not work, please see https://github.com/pre-commit/pre-commit/issues/1445 -- I am still looking for a workaround for this issue. Maybe you can find one?

---

I don't see this specific option set in that issue snippets. Also, I think I use this elsewhere and it works.